articles/user-guide/subaccounts.md

```diff
@@ -166,6 +169,10 @@ The master account may send as the subaccount with the transmission API by inclu
 
 The master account may submit traffic on behalf of the subaccount SMTP by providing the subaccount's API key with an SMTP grant in the TLS auth-password field. **Note**: The master account can send using a master account API key by appending the standard "SMTP_Injection" user with the subaccount ID. Please refer to [the documentation](https://developers.sparkpost.com/api/?_ga=1.144252341.1033930248.1481562971#/introduction/smtp-relay-endpoints) for specifics on how to perform this operation.
 
+**Event Webhooks**
+
+The master account may create a webhook as the subaccount with the webhooks API by including the X-MSYS-SUBACCOUNT HTTP header. If the header is not present, the webhook will be executed as the master account.
```

Comment:
Rewording suggestion - 

"... the webhook will be created in the master account."